### ISSUE TITLE
Ability to translate the caption for "Done" button.

### ISSUE THREAD

**Issue by movisis** [stormlion227/PhotoBrowser.Forms#7]
I'm developing an app in Portuguese, and wanted to have the ability to change the title for that button. Could be changed to "OK", or make a property to put whatever we wanted.

**Comment by candidodmv** [comment]
I think the same thing, would be very useful have any way to internationalization messages and texts.

**Comment by E-Coutret** [comment]
Still no solution to translate the button ?

**Comment by candidodmv** [comment]
No. I think that author have been so busy, because there are no one interaction by him
